src/components/screens/login/networkSelector/customNode/editMode.js

```diff
@@ -22,6 +22,16 @@ const validateNode = async (address) => {
   }
 };
 
+const removeTrailingSlash = (input) => {
+  const bypassStrings = ['http:/', 'http://', 'https:/', 'https://'];
+
+  if (input.charAt(input.length - 1) !== '/' || bypassStrings.includes(input)) {
+    return input;
+  }
+
+  return input.substring(0, input.length - 1);
+};
```

Comment:
Why not use a regex?

```suggestion
/**
 * Removes the trailing slash
 *
 * @param {string} url - A URL that might end in a slash
 * @returns {string} A URL without a trailing slash
 */
const removeTrailingSlash = url => url.replace(/\/$/, '');
```